src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -32,9 +40,13 @@ function LineVis(): JSX.Element {
           showGrid,
         }}
         ordinateConfig={{
-          dataDomain: extendDomain(dataDomain, 0.01),
+          dataDomain: extendDomain(
+            dataDomain,
+            0.05,
+            scaleType === ScaleType.Log
+          ),
```

Comment:
I wonder if the domain extension could be put in the `setDataDomain` from the store as well...